servicetalk-loadbalancer-experimental-provider/src/main/java/io/servicetalk/loadbalancer/experimental/DefaultHttpLoadBalancerProvider.java

```diff
@@ -38,25 +36,18 @@ public class DefaultHttpLoadBalancerProvider implements HttpProviders.SingleAddr
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultHttpLoadBalancerProvider.class);
 
-    private final DefaultLoadBalancerProviderConfig config;
-
     public DefaultHttpLoadBalancerProvider() {
-        this(DefaultLoadBalancerProviderConfig.INSTANCE);
-    }
-
-    // exposed for testing
-    DefaultHttpLoadBalancerProvider(final DefaultLoadBalancerProviderConfig config) {
-        this.config = requireNonNull(config, "config");
     }
 
     @Override
     public final <U, R> SingleAddressHttpClientBuilder<U, R> newBuilder(U address,
                                                                   SingleAddressHttpClientBuilder<U, R> builder) {
         final String serviceName = clientNameFromAddress(address);
+        final DefaultLoadBalancerProviderConfig config = DefaultLoadBalancerProviderConfig.instance();
```

Comment:
Should we log (at debug/info level) the current snapshot of the config for the current `address`?

Reply:
Debug should be fine.